pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "qdrant-client"
-version = "1.4.0"
+version = "1.6.9"
```

Comment:
please exclude it from the PR, otherwise there will be a conflict on merging dev -> master